examples/tutorials/flow-deploy/create-service-with-flow/resources.txt

```diff
@@ -0,0 +1 @@
+examples/tutorials/flow-deploy/create-service-with-flow
```

Comment:
too many txt files with this change. a bit ugly
![image](https://github.com/microsoft/promptflow/assets/47586720/63df0bb5-719b-4c74-8523-132a663b8f1b)

Reply:
yes... if we can accept inconsistency between notebook and readme, how about adding a folder like `examples/tutorials/resources/`, and we can host these txts there like `examples/tutorials/resources/run-management.txt`

Reply:
for readme, it looks like below for now:

![image](https://github.com/microsoft/promptflow/assets/38847871/205a772d-bac0-42b4-a64a-47e41f79c216)

it's easier for me to accept this

Reply:
another proposal is to explicitly add resources in notebook/readme with special mark (e.g. `# resources`), but it's not very user friendly...

Reply:
offline synced, will integrate this into notebook/markdown and not visible when rendering.

Reply:
updated using metadata, may take a look

Reply:
btw, notebook will not render metadata, but GitHub will...

![image](https://github.com/microsoft/promptflow/assets/38847871/54babd08-aa02-4be2-9062-f3646a49cf61)